compiler/modules/types.regex/tests/data/regexInclusionTests.json

```diff
@@ -15,7 +15,5 @@
 
     ["<", "(AC)*F", "(AB*C)*F"],
     ["<", "(AC)*F", "(A(B)*C)*F"],
-    ["=", "(AB*C)*F", "(A(B)*C)*F"],
-
-    ["<", "(RgeRgF(b((OS)*cDVB)*H)*kYLy)*W", "(RgeRgF(b(((glPrQ)*OS)*cDVB)*H)*kYLy)*W"]
```

Comment:
Why is this PR removing tests?

Reply:
fixed (without #1092 this test will run for a very long time but I don't think it caused by this change)